CryptoTechReminderSystem.Test/Gateway/HarvestGatewayTests.cs

```diff
@@ -99,7 +99,7 @@ public void Setup()
 
             private void SetUpTimeSheetApiEndpoint(string dateFrom, string dateTo)
             {
-                var json = File.ReadAllText(
+                var jsonPageOne = File.ReadAllText(
                     Path.Combine(
                         AppDomain.CurrentDomain.BaseDirectory,
                         "../../../Gateway/HarvestTimeEntriesApiEndpoint.json"
```

Comment:
This could be renamed to `HarvestTimeEntriesApiEndpointPageOne.json`.

Reply:
Resolved in https://github.com/chubberlisk/crypto_tech/pull/33/commits/138fd1a88028a256575f2eb1b560fd8e96ec1256. File has also been renamed.